setup.py

```diff
@@ -24,7 +24,7 @@ def get_long_description():
                  license='LGPL-3.0',
 
                  use_scm_version=True,
-                 install_requires=['numpy>=1.20,<2.0', 'segyio', 'zfpy', 'psutil', 'click'],
+                 install_requires=['numpy>=1.22.2,<2.0.0', 'segyio', 'zfpy', 'psutil', 'click'],
```

Comment:
Here we actually need the 1.20 requirement for some python distributions

Reply:
As long as all are aligned it is fine.